.github/workflows/iam-handle.yml

```diff
@@ -79,6 +79,6 @@ jobs:
       - name: 'Run AOD CLI'
         env:
           DURATION: ${{ env.LABELED_DURATION || '2h' }}
```

Comment:
It does not allow me to quote this one because the '2h' already has quote.An alternative is to set a default duration env variable, and do DURATION: ${{ env.LABELED_DURATION || env.DEFAULT_DURATION }}, will check if there is a better option before doing that.

Reply:
No, it gives me --> Unexpected symbol: '"2h"'. Located at position 25 within expression: env.LABELED_DURATION || "2h"

Reply:
I am keeping it as is, since DURATION: '${{ env.LABELED_DURATION || env.DEFAULT_DURATION }} ' needs me to define DEFAULT_DURATION in earlier steps. Feel like the current way is the cleanest.

Reply:
Does this work? Define `env.DEFAULT_DURATION` at the top might actually be clearer IMO. And you can avoid introducing an "outlier" here (not quoting).

Reply:
Top level should work.

Reply:
Done.